wdio.conf.js

```diff
@@ -297,8 +297,9 @@ exports.config = {
   /**
    * Function to be executed after a test (in Mocha/Jasmine).
    */
-  // afterTest: function(test, context, { error, result, duration, passed, retries }) {
-  // },
+  afterTest: function(test, context, { error, result, duration, passed, retries }) {
```

Comment:
My guess is that the commented out stuff is here to show folks using this tool what the function parameters look like. That's useful, and I'd like to keep that around, but as this is written none of those arguments are used in the function. This could be changed to the below, to keep the method signature, but only use the arguments we actually care about

```suggestion
  // afterTest: function(test, context, { error, result, duration, passed, retries }) {
  afterTest: function() {
```